unittests/linuxliketests.py

```diff
@@ -1352,7 +1352,7 @@ def test_link_arg_fullname(self):
         #   before v0.28, Libs flags like -Wl will not kept in context order with -l flags.
         #   see https://gitlab.freedesktop.org/pkg-config/pkg-config/-/blob/master/NEWS
         pkgconfigver = subprocess.check_output([PKG_CONFIG, '--version'])
-        if b'0.28' > pkgconfigver:
+        if pkgconfigver < b'0.28':
```

Comment:
?

Reply:
I'm going to guess this is wanting to fix the [yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions)